media-gfx/maim/Manifest

```diff
@@ -0,0 +1,3 @@
+DIST maim-3.4.47.tar.gz 46799 SHA256 fb12524cc05f7f4ff46fe40b951334c6dbc274e50b6d0ea08185f555ca3cd74d SHA512 4b21f8ad8f9f2f19693d5c489f5e815133e1b8308653c52934cdb8a3d8ae38732733a5ffb5527b44db49f59c3818a9bc83840ced5db71807a68277d0dcc03392 WHIRLPOOL de21bb02c2923fd158cc691df09ff9a55121206f2f491a6b4bcf080b7b56032822f7ac4bad98028294f5f6e6e103fb393db3933dc3a7efcd383db6602d2d17d9
+EBUILD maim-3.4.47.ebuild 557 SHA256 dd2828950e6b4f89fa0feceaa965dfd15cdb3b50c00c802a0c9858bf9b2c6d53 SHA512 294753541b6f6264120afb9d30c972ddeb2aa4a0294707f1aef7294cfbfe4626eb78e8ab1545eafc5fc4f648ca7942b1723ab5ac80484bb4ed0e9894fd1f4492 WHIRLPOOL e7080c98dbe30e8e8a8a958d02145f6a6871cf851d07564b55dba600aa36e84a47f12531b27e680bcd8b8902a07021a43a29a63a66975ce2aded45711756ff8b
+MISC metadata.xml 559 SHA256 2fb690bc6edf9ec1be0f5a8341159e0b1c9a8251eb3031841650e111e289267f SHA512 9204e50059992d04ef1d117c74621bbf4e4d20299a3e8af97483aa7f36daff18dc6015d9656845ff60085d737597d6b5e3a1e490ce60e292f906fd6e399e2af7 WHIRLPOOL 04b1bd5adb2b4dde97b50e830186fbfb4391a82a186ff0e3d515bcc2c5f2211ca0eaa8e48f170c73922148504547fcfc64969b58ca92a0b07b27f8d1dd8d86cd
```

Comment:
this is a fat manifest, the normal gentoo tree uses thin manifests. Please fix these